src/core/lib/slice/slice.cc

```diff
@@ -407,21 +309,21 @@ grpc_slice grpc_slice_split_tail_maybe_ref(grpc_slice* source, size_t split,
       tail.data.inlined.length = static_cast<uint8_t>(tail_length);
       memcpy(tail.data.inlined.bytes, source->data.refcounted.bytes + split,
              tail_length);
-      source->refcount = source->refcount->sub_refcount();
+      source->refcount = source->refcount;
```

Comment:
This is assigning the variable to itself, which is presumably a no-op (or maybe UB? I don't remember).  Either way, I assume that's not what is actually intended here?

Reply:
It kind of is intended (it's not UB, it's a no-op) -- but more an indicator that this line isn't needed anymore.